packages/gatsby/src/utils/webpack.config.js

```diff
@@ -188,8 +184,10 @@ module.exports = async (
       // optimizations for React) and what the link prefix is (__PATH_PREFIX__).
       plugins.define({
         ...processEnv(stage, `development`),
-        __PATH_PREFIX__: JSON.stringify(
-          program.prefixPaths ? store.getState().config.pathPrefix : ``
+        __BASE_PATH__: JSON.stringify(program.prefixPaths ? pathPrefix : ``),
+        __PATH_PREFIX__: JSON.stringify(program.prefixPaths ? publicPath : ``),
+        __ASSET_PREFIX__: JSON.stringify(
+          program.prefixPaths ? assetPrefix : ``
```

Comment:
I'm not sure where to document this, but I feel like it would be useful to define differences between those. To my understanding plugins (for now) should only ever use `__PATH_PREFIX__`? `__ASSET_PREFIX__` seems not used ATM anywhere - would it make sense not to define it at all?

Reply:
Correct! `__BASE_PATH__` should be used for what was `__PATH_PREFIX__`, and `__ASSET_PREFIX__` isn't really used. I'll look for a best place to document this!